.eslintrc

```diff
@@ -0,0 +1,102 @@
+{
+  "parser": "babel-eslint",
+  "extends": [
+    "standard",
+    "eslint:recommended",
+    "plugin:react/recommended"
+  ],
+  "plugins": [
+    "flow-vars",
+    "react",
+    "jsx-a11y",
+    "import",
+    "babel",
+    "immutable"
+  ],
+  "env": {
+    "browser" : true
+  },
+  "rules": {
+    "semi" : [2, "never"],
```

Comment:
Maybe. My thinking is, it's better to be explicit. But more importantly, using semicolons is the way I'm used to doing it, and I don't see enough gain here to change my habits :)